packages/nhost-js/src/utils/helpers.ts

```diff
@@ -19,7 +19,9 @@ export function urlFromParams(
   }
 
   if (backendOrSubdomain.subdomain !== undefined && backendOrSubdomain.subdomain.match(LOCALHOST)) {
-    return `http://${backendOrSubdomain.subdomain}/v1/${service}`
+    return backendOrSubdomain === 'localhost'
+      ? `http://localhost:1337/v1/${service}`
+      : `http://${backendOrSubdomain.subdomain}/v1/${service}`
```

Comment:
For developers who are currently using `localhost:1337` this update will break their code.

I think we should allow for `localhost:<port>` here. Something like this:


```js
return backendOrSubdomain === 'localhost'
      ? `http://localhost:1337/v1/${service}`
      : backendOrSubdomain.startsWith('localhost:') (or regex) 
        ? `http://${backendOrSubdomain}/v1/${service}` 
        : `http://${backendOrSubdomain.subdomain}/v1/${service}`
```